bridge/engine/VideoForwarderRewriteAndSendJob.cpp

```diff
@@ -65,26 +64,27 @@ void VideoForwarderRewriteAndSendJob::run()
     auto rtpHeader = rtp::RtpHeader::fromPacket(*_packet);
     if (!rtpHeader)
     {
+        assert(false); // should have been checked multiple times
         return;
     }
 
-    bool isRetransmittedPacket = false;
     if (_outboundContext.rtpMap.format == RtpMap::Format::VP8RTX)
     {
-        isRetransmittedPacket = true;
+        logger::warn("%s rtx packet should not reach rewrite and send. ssrc %u, seq %u",
```

Comment:
See https://perzoinc.atlassian.net/browse/RTC-13340
rtx should be converted to (out of order rtp packets). Payload type should never be rtx once processed in RtxReceiveJob